packages/RichTextEditing-Tests.package/RichTextStructureEditorTests.class/instance/displayedStructuresEqualInitialStructuresOf..st

```diff
@@ -0,0 +1,4 @@
+utilities
+displayedStructuresEqualInitialStructuresOf: aRichTextTemplate
+
+	^ (self structureEditorView findByName: 'structureField') getList = aRichTextTemplate initialStructureIdentifiers
```

Comment:
I also don't unterstand what this does/ why it is necessary

Reply:
It checks if, after selecting a new teplate, the displayed structures in the structure editor are identical to the initial structures of the new template

Reply:
can we not just use ```self textStructures```?

Reply:
Maybe, but we are not testing that they are displayed then, are we?

Reply:
This is true but it should be the same. Do we want to test if data and view match each other or the general function?

Reply:
Since it is an acceptance test, we should perhaps prefer testing the view as done here, because the customer does not care about the data directly